pywwt/annotation.py

```diff
@@ -230,3 +231,44 @@ def _on_trait_change(self, changed):
             changed['new'] = changed['new'].value
 
         super(Line, self)._on_trait_change(changed)
+
+class CircleCollection():
+    
+    def __init__(self, parent, points, **kwargs): # save args, make first circles
+        if len(points) <= 1e4: # subject to change
+            self.points = points
+        else:
+            raise IndexError('too many points') # correct error?
```

Comment:
Maybe change this error to 'For performance reasons, only 10000 annotations can be added in one go at this time'